src/glmtools.jl

```diff
@@ -1,4 +1,4 @@
-using Base.Cartesian
+#using Base.Cartesian
```

Comment:
I will do so.  I just wanted to check that commenting it out didn't cause a failure in the tests.